### ISSUE TITLE
deactivate_requires_grad does not fix the model!

### ISSUE THREAD

**Issue by AmirooR** [lightly-ai/lightly#1438]
Hi I just noticed that in the examples the backbone is fixed by this line: https://github.com/lightly-ai/lightly/blob/master/lightly/models/utils.py#L168 

It makes the requires_grad to variables to False. However, the batchnorm statistics (running_mean, running_var, etc.) are still being updated. You should make the batchnorm modules to eval mode if fixing the backbone models is desired.

**Comment by guarin** [comment]
Hi, thanks for brining this up! This is correct and whether batch norm statistics should be updated or not depends on the model. As far as I know most models do not set batchnorm modules to eval, for example [MoCo](https://github.com/facebookresearch/moco/blob/1999df86de600027cd2702cbd7cd7b7d2a53559d/moco/builder.py), [DINO](https://github.com/facebookresearch/dino/blob/main/main_dino.py), and [MSN](https://github.com/facebookresearch/msn/blob/main/src/msn_train.py) all leave the batchnorm layers in train mode.

**Comment by guarin** [comment]
Closing this for now, please reopen if you have more questions :)
